### PR TITLE
added Scrolldir library

### PR DESCRIPTION
**Pull request by surajhanchinal**
Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * License and its reference:
 * GitHub / Bitbucket popularity (required):
   - Count of watchers:
   - Count of stars:
   - Count of forks:
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
hi @surajhanchinal
please remember to fulfill the lib info in the init comment, which will help us review your PR more quickly, thanks!

File: ajax/libs/scrolldir/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "scrolldir",
+  "filename": "dist/scrolldir.min.js",
+  "version": "1.1.5",
+  "description": "Vertical scroll direction in CSS",
+  "main": "dist/scrolldir.min.js",
```
Comment (maruilian11) [inline ajax/libs/scrolldir/package.json]: please remove `main`.

File: ajax/libs/scrolldir/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "scripts": {
```
Comment (maruilian11) [inline ajax/libs/scrolldir/package.json]: please remove `scripts`.

File: ajax/libs/scrolldir/package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    "scroll",
+    "intent",
+    "scrollTop",
+    "position",
+    "javascript",
+    "window"
+  ],
+  "author": "Patrick Fisher <patrick@pwfisher.com>",
+  "license": "MIT",
+  "bugs": {
```
Comment (maruilian11) [inline ajax/libs/scrolldir/package.json]: please remove `bugs`.

File: ajax/libs/scrolldir/package.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    "url": "https://github.com/dollarshaveclub/scrolldir.git/issues"
+  },
+  "homepage": "https://github.com/dollarshaveclub/scrolldir.git",
```
Comment (maruilian11) [inline ajax/libs/scrolldir/package.json]: please change to use `"https://dollarshaveclub.github.io/scrolldir"`, thanks.

File: ajax/libs/scrolldir/package.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  "devDependencies": {
```
Comment (maruilian11) [inline ajax/libs/scrolldir/package.json]: please also remove `devDependencies`, thanks.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!}

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!}

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by maruilian11** [comment]
@surajhanchinal If you don't have much time to send this PR, I'm willing to send it for you. Thanks.

**Comment by sufuf3** [comment]
@PeterDaveHello I think this PR can close, because we hosted it.
As same with #10657 
https://cdnjs.com/libraries/scrolldir

**Pull request closed without merging**